TeensyDmx.h

```diff
@@ -56,6 +58,10 @@ struct RdmData
 static_assert((sizeof(RdmData) == 255),
               "Invalid size for RdmData struct, is it packed?");
 
+typedef void (*rdmControllerCallback)(CallbackStatus, RdmData*);
```

Comment:
Prefer using over typedef in C++ now.

Reply:
Done, I think, it compiles, as yet untested.